packages/dataviews/src/normalize-fields.ts

```diff
@@ -1,10 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import type { Field } from './types';
+
 /**
  * Apply default values and normalize the fields config.
  *
- * @param {Object[]} fields Raw Fields.
- * @return {Object[]} Normalized fields.
+ * @param fields Fields config.
```

Comment:
Isn't the type missing here?

Reply:
now, in typescript, it's not needed because it's actually defined as part of the code.

Reply:
Oh, I hadn't noticed the file had been switched to TS. Do we need to switch our code to TypeScript for us to use the TS types in the docs? I thought we could use them in the JSDoc directly.

Reply:
@oandregal you can use them in JSDocs. I switched to typescript extension because I didn't want to list all the JS files that are fully type-checked in tsconfig file. the tsconfig file only targets all "ts" files for now.